test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -23,6 +23,23 @@ namespace Test.Utility.Signing
 {
     public static class SigningTestUtility
     {
+        private static string _testCertPassword = Guid.NewGuid().ToString();
+
+        /// <summary>
+        /// Creates a copy of an X509Certificate2 by exporting the original certificate into a pfx file.
+        /// </summary>
+        /// <param name="originalCertificate">X509Certificate2 to be copied.</param>
+        /// <returns>X509Certificate2 copy of the originalCertificate.</returns>
+        public static X509Certificate2 CopyCertificate(X509Certificate2 originalCertificate)
+        {
+            var copiedCertificate = new X509Certificate2(
```

Comment:
I think this can be simplified significantly.

    return new X509Certificate2(originalCertificate);

This [clones the underlying certificate context](https://referencesource.microsoft.com/#mscorlib/system/security/cryptography/x509certificates/x509certificate.cs,195).  Disposing the original certificate [won't dispose the underlying context](https://referencesource.microsoft.com/#mscorlib/system/security/cryptography/x509certificates/x509certificate.cs,568).

In fact, you can just inline this at call sites:

    using (var clonedCertificate = new X509Certificate2(_trustedTestCert.Source.Cert))